# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit/BaselineIC.h
# Commit: bde81f964234
# Full Hash: bde81f9642344b7bc2fab62380771c670138aaac
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-08-08 21:45:31
# Description:
#   Bug 1572051 - Add TypeMonitorMagicValue and use it in DoTypeMonitorFallback and TypeMonitorResult. r=iain
#   
#   We did a bailout from Ion with a MagicValue(JS_OPTIMIZED_OUT) on top of the stack, then passed that
#   value to TypeMonitorResult and asserted in addMonitorStubForValue because we expected a different
#   kind of magic value.
# ==============================================================================

diff -r 7d7830b79440 -r bde81f964234 js/src/jit/BaselineIC.h
--- a/js/src/jit/BaselineIC.h	Thu Aug 08 12:23:20 2019 +0300
+++ b/js/src/jit/BaselineIC.h	Wed Aug 07 19:04:38 2019 +0000
@@ -1048,9 +1048,6 @@
 
  public:
   MOZ_MUST_USE bool initMonitoringChain(JSContext* cx, JSScript* script);
-  MOZ_MUST_USE bool addMonitorStubForValue(JSContext* cx, BaselineFrame* frame,
-                                           StackTypeSet* types,
-                                           HandleValue val);
 
   ICTypeMonitor_Fallback* maybeFallbackMonitorStub() const {
     return fallbackMonitorStub_;
